woocommerce/woocommerce-gutenberg-products-block
# Unify Cart headers

Fixes #2614.

#### Accessibility


### Screenshots
_Before:_
![imatge](https://user-images.githubusercontent.com/3616980/83630633-6ff48b00-a59c-11ea-913c-a5bf66e75349.png)

_After:_
![imatge](https://user-images.githubusercontent.com/3616980/83630607-6408c900-a59c-11ea-9790-3168b3fb476a.png)

### How to test the changes in this Pull Request:

Open the _Cart_ block and verify you see a heading in the form of `X products - X items` and that the numbers are correct.

### Changelog

> The Cart block titles have been merged into one.
